plugin/storage/memory/memory_test.go

```diff
@@ -224,18 +227,42 @@ func TestStoreGetServices(t *testing.T) {
 	})
 }
 
-func TestStoreGetOperationsFound(t *testing.T) {
+func TestStoreGetAllOperationsFound(t *testing.T) {
 	withPopulatedMemoryStore(func(store *Store) {
-		operations, err := store.GetOperations(context.Background(), testingSpan.Process.ServiceName)
+		assert.NoError(t, store.WriteSpan(testingSpan))
+		assert.NoError(t, store.WriteSpan(childSpan1))
+		assert.NoError(t, store.WriteSpan(childSpan2))
+		assert.NoError(t, store.WriteSpan(childSpan2_1))
+		operations, err := store.GetOperations(context.Background(),
+			&spanstore.OperationQueryParameters{ServiceName: childSpan1.Process.ServiceName})
+		assert.NoError(t, err)
+		assert.Len(t, operations, 3)
+		assert.EqualValues(t, childSpan1.OperationName, (*operations[0]).Name)
+	})
+}
+
+func TestStoreGetServerOperationsFound(t *testing.T) {
+	withPopulatedMemoryStore(func(store *Store) {
+		assert.NoError(t, store.WriteSpan(testingSpan))
+		assert.NoError(t, store.WriteSpan(childSpan1))
+		assert.NoError(t, store.WriteSpan(childSpan2))
+		assert.NoError(t, store.WriteSpan(childSpan2_1))
+		operations, err := store.GetOperations(context.Background(),
+			&spanstore.OperationQueryParameters{
+				ServiceName: childSpan1.Process.ServiceName,
+				SpanKind:    "server",
+			})
 		assert.NoError(t, err)
 		assert.Len(t, operations, 1)
```

Comment:
it's much easier to read if you just used assert.Equal and pass the expected values in a structured form, e.g.
```
expected := []spanstore.Operation{{Name: .., Kind: ...}}
assert.Equal(t, expected, operations)
```